tensor2tensor/rl/trainer_model_free.py

```diff
@@ -32,11 +43,38 @@
 except:  # pylint: disable=bare-except
   pass
 
+@registry.register_hparams
+def pong_model_free():
+  hparams = tf.contrib.training.HParams(
+    epochs_num=4,
+    eval_every_epochs=2,
+    num_agents=10,
+    optimization_epochs=3,
+    epoch_length=30,
+    entropy_loss_coef=0.003,
+    learning_rate=8e-05,
+    optimizer="Adam",
+    policy_network=rl_models.feed_forward_cnn_small_categorical_fun,
+    gae_lambda=0.985,
+    num_eval_agents=1,
+    max_gradients_norm=0.5,
+    gae_gamma=0.985,
+    optimization_batch_size=4,
+    clipping_coef=0.2,
+    value_loss_coef=1,
+    save_models_every_epochs=False)
+  hparams.add_hparam(
+    "environment_spec",
+    gym_problems.standard_atari_env_spec("PongNoFrameskip-v4"))
+  hparams.add_hparam(
+    "environment_eval_spec",
+    gym_problems.standard_atari_env_eval_spec("PongNoFrameskip-v4"))
+
+  return hparams
 
 def main(_):
   hparams = trainer_lib.create_hparams(FLAGS.hparams_set, FLAGS.hparams)
-  rl_trainer_lib.train(hparams, FLAGS.problem, FLAGS.output_dir)
-
+  rl_trainer_lib.train(hparams, FLAGS.output_dir, FLAGS.output_dir)
```

Comment:
line accidentally changed?

Reply:
It is a bug fix.

Reply:
Ah! Just looked at the method signature now.